service/internal/acl/acl.go

```diff
@@ -196,18 +197,28 @@ func UserFromSystem(cfg *config.Config, username string) *AuthenticatedUser {
 }
 
 func buildUserAcls(cfg *config.Config, user *AuthenticatedUser) {
+Lists:
 	for _, acl := range cfg.AccessControlLists {
 		if slices.Contains(acl.MatchUsernames, user.Username) {
 			user.Acls = append(user.Acls, acl.Name)
 			continue
 		}
 
-		if slices.Contains(acl.MatchUsergroups, user.Usergroup) {
+		// handle multiple usergroups - groups will be separated by a space
+		if hasGroupsMatch(acl.MatchUsergroups, user.Usergroup) {
 			user.Acls = append(user.Acls, acl.Name)
-			continue
+			continue Lists
```

Comment:
@iamnoah Is the labelled loop here necessary? There is only one level of loop, and the condition above (`slices.Contains`) doesn't use the label?

Reply:
No, you're right, I had it there before extracting a function and forgot to clean it up. Gone now.